src/Processors/QtiV2/In/Interactions/ChoiceInteractionMapper.php

```diff
@@ -90,4 +97,47 @@ private function buildOptions(SimpleChoiceCollection $simpleChoices)
         }
         return $options;
     }
+
+    private function buildFeedbackMetadata(SimpleChoiceCollection $simpleChoices)
```

Comment:
Not a major concern, but it would be great to include a simple method DocBlock (in phpDocumentor style) to document things like return type / value, any exceptions, description of parameters etc. Ideally we should add this for all new methods going forward.

For example:
```
/**
 * [Some one-line description of the method]
 *
 * @param SimpleChoiceCollection $simpleChoices - [Description of param]
 *
 * @returns [ReturnType] - [Short description of the return value]
 */
```

Reply:
Done